src/Symfony/Component/Form/Extension/Core/Type/FormFlowNavigatorType.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Flow\FormFlowCursor;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * A navigator type that defines default actions to interact with a form flow.
+ *
+ * @author Yonel Ceruto <open@yceruto.dev>
+ */
+class FormFlowNavigatorType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder->add('back', FormFlowActionType::class, [
```

Comment:
what bout "previous" ? generally it goes with "next"
or rename other "forward" ?

Reply:
I don’t have a strong opinion on that. I personally prefer the shorter version, and users can change it using the `label` option anyway. Let’s see what others think.

Reply:
I have no preference between "prev/next" and "back/forward" but it does feel weird to have both, so i would agree with @94noni here.

In multiple steps form i do believe "next" is some de-facto standard, so "previous" would be the one ?

Reply:
Ok, I will rename the default name/label to "Previous" instead of "Back"

Reply:
Updated!